laravel-app/resources/views/profile/show.blade.php

```diff
@@ -5,14 +5,16 @@
 @include('partials._header')
 
 @php
-	$authUser = auth()->user();
+    $authUser = auth()->user();
+    $imagePath = $authUser && $authUser->media->isNotEmpty() ? 'storage/' . $authUser->media->last()->file_path : 'assets/images/user-solid.svg';
 @endphp
```

Comment:
Transfer this to your controller.

Reply:
Already transferred it sir to controller